### ISSUE TITLE
The difference between torch.ops.aten.convolution.default and  torch.ops.aten.conv2d.default 

### ISSUE THREAD

**Issue by TingfengTang** [pytorch/pytorch#137885]
what's the difference between those two ops after using torch.export.export.
i saw the example given in [torch.export.export](https://pytorch.org/docs/stable/export.html)is torch.ops.aten.convolution.default,but when i export my own model,it shows the aten op i used is  torch.ops.aten.conv2d.default and it missed some parameters.

cc @ezyang @chauhang @penguinwu @avikchaudhuri @gmagogsfm @zhxchen17 @tugsbayasgalan @angelayi @suo @ydwu4

**Comment by angelayi** [comment]
`aten.conv2d` is a "higher level" representation of convolution and will later decompose to `aten.convolution`. If you look at the registration of all aten functions, [`aten.conv2d`](https://github.com/pytorch/pytorch/blob/0eb425a56304c35665eb60aae6b0450dc7fd2ef7/aten/src/ATen/native/native_functions.yaml#L1712-L1714) is marked as dispatching to `CompositeImplicitAutograd` whereas [`aten.convolution`](https://github.com/pytorch/pytorch/blob/0eb425a56304c35665eb60aae6b0450dc7fd2ef7/aten/src/ATen/native/native_functions.yaml#L1673-L1677) is marked as `CompositeExplicitAutograd`. CompositeImplicitAutograd op implementations are at a higher level as they can also support autograd, and later get decomposed to CompositeExplicitAutograd operators.

Currently `torch.export.export` returns operators at the CompositeImplicitAutograd layer, but previously (at the time when writing the doc) it would return operators at the CompositeExplicitAutograd layer. You can reproduce the same behavior and get to the CompositeExplicitAutograd layer by calling `torch.export.export(...).run_decompositions()`.

What do you mean by it's missing some parameters? Do you have a repro?

**Comment by TingfengTang** [comment]
thank you for your reply.

the code i used is :
```
import torch
import torch.nn as nn

class SimpleCNN(nn.Module):
    def __init__(self):
        super(SimpleCNN, self).__init__()
        self.conv1 = nn.Conv2d(in_channels=3, out_channels=16, kernel_size=3, stride=1, padding=(1,1),bias=False,padding_mode='zeros')

    def forward(self, x):
        x = self.conv1(x)
        return x
torch.manual_seed(0)
model=SimpleCNN().eval()
input=torch.randn(3,3,224,224)
model_ep:torch.export.ExportedProgram =torch.export.export(model,(input,))
print(model_ep)
```
the result is:
```
ExportedProgram:
    class GraphModule(torch.nn.Module):
        def forward(self, p_conv1_weight: "f32[16, 3, 3, 3]", x: "f32[3, 3, 224, 224]"):
            # File: /data/tangtingfeng/work/torch2Gtnn/test_Conv2d.py:19 in forward, code: x = self.conv1(x)
            conv2d: "f32[3, 16, 224, 224]" = torch.ops.aten.conv2d.default(x, p_conv1_weight, None, [1, 1], [1, 1]);  x = p_conv1_weight = None
            return (conv2d,)
            
Graph signature: ExportGraphSignature(input_specs=[InputSpec(kind=<InputKind.PARAMETER: 2>, arg=TensorArgument(name='p_conv1_weight'), target='conv1.weight', persistent=None), InputSpec(kind=<InputKind.USER_INPUT: 1>, arg=TensorArgument(name='x'), target=None, persistent=None)], output_specs=[OutputSpec(kind=<OutputKind.USER_OUTPUT: 1>, arg=TensorArgument(name='conv2d'), target=None)])
Range constraints: {}
```
i guess the two lists pamameters [1,1] of torch.ops.aten.conv2d.default  are correspond to stride and padding. where is the other pamameters like dilation and groups Or they are default pameters,if so ,how can i find the declaration of the default values.

my purpose is to extract the parameters when i transverse through all the nodes of the graph.

thanks.

**Comment by angelayi** [comment]
You can do the following to get the default arguments:
```
for arg in torch.ops.aten.conv2d.default._schema.arguments:
   print(arg.name, arg.default_value) 
```
